nbd/src/util.cpp

```diff
@@ -278,6 +280,37 @@ int get_mapped_info(int pid, NBDConfig *cfg) {
     return 0;
 }
 
+int check_mount_status(const NBDConfig *cfg) {
+    std::ifstream ifs;
+    ifs.open("/proc/mounts", std::ifstream::in);
```

Comment:
You can merge this as one line code `std::ifstream ifs("/proc/mounts", std::ifstream::in);`

Reply:
> td::ifstream ifs("/proc/mounts", std::ifstream::in);

done.